src/hal/display.cpp

```diff
@@ -70,7 +70,8 @@ void OswHal::setupDisplay() {
     if(!this->canvas)
         this->canvas = new Arduino_Canvas_Graphics2D(DISP_W, DISP_H, tft);
 
-    this->canvas->begin(0);
+    this->canvas->begin();  // use default speed and default SPI mode
+    delay(45); // avoid flickering
     this->displayOn();
```

Comment:
any references? could you add a comment explaining this?

(because I recall removing anti-flicker code before, because there was no comment explaining it and I was not observing it)